orttraining/orttraining/core/graph/gradient_builder_base.h

```diff
@@ -329,6 +339,8 @@ class GradientBuilderBase {
   std::unordered_set<std::string> gradient_outputs_;
 
   const logging::Logger& logger_;
+
+  std::unordered_map<std::string, int>& input_arg_ref_count_;
```

Comment:
>int [](http://example.com/codeflow?start=34&length=3)

this doesn't need to be an Int, bool is sufficient, as we only care about stashed or not.

Reply:
Agreed.